.github/workflows/linux-build.yml

```diff
@@ -0,0 +1,280 @@
+name: ODBC Driver for Linux
+
+on:
+  push:
+    paths:
+      - '.github/workflows/win-build.yml'
```

Comment:
`linux-build.yml`?

Reply:
Addressed in https://github.com/Bit-Quill/arrow/pull/7#discussion_r1452813092